CSharp/DevVmPowershell/Helpers.Tests.Integration/TestConstants.cs

```diff
@@ -3,10 +3,11 @@
 	public class TestConstants
 	{
 		public const string TEST_APPLICATION_NAME = "Production";
-		public const string RELATIVITY_INSTANCE_NAME = "172.28.87.69";
-		public const int RELATIVITY_WORKSPACE_ID = 1017606;
+		public const string RELATIVITY_INSTANCE_NAME = "172.17.217.154";
+		public const int RELATIVITY_WORKSPACE_ID = 1017323;
 		public const string RELATIVITY_INSTANCE_WEB_SERVICE_URL = "http://" + RELATIVITY_INSTANCE_NAME + "/relativitywebapi/";
 		public const string RELATIVITY_ADMIN_USER_NAME = "relativity.admin@relativity.com";
 		public const string RELATIVITY_ADMIN_PASSWORD = "Test1234!";
+		public const string APPLICATION_FILE_PATH = "C:\\Users\\aaron.gilbert\\Downloads\\Relativity_Smoke_Test_2.1.0.1.rap";

```

Comment:
can we add the rap file to the repository and point to that file? So that next time someone clones this repo, the file will be there.